README.md

```diff
@@ -104,70 +176,132 @@ Username:
 Password:
 > enter your github token
 
-Click the "*connect*" button at the top of the page to save the connection
+Click the `Connect` button at the top of the page to save the connection
 
-If the information was entered correctly the connection status will be *Successful*
+If configured correctly, the connection status will show as *Successful*
 
```

Comment:
The confluence page has something here about enabling argo ingress. Is this desired to keep? The text is


The argo ingress is not enabled by default, to change this, navigate to argo/chart/values.yaml and modify the following lines (Note, if you're using vim, an easy way to navigate to these lines is :<line number> )

At (or around) line 2017, change **"enabled: false"** to **"enabled: true"**

At (or around) line 2033, change - **[]** under "hosts: " to - **argo.10.96.30.9.nip.io**

Reply:
yeah that's a bad habit that I didn't quash soon enough. best practice is to only expose argo over port-forward